File: JavaScript/01-Number-Programs/palindrome.js
function isPalindrome(num) {
  let originalVal = num;
  let reverse = 0;

  while (num > 0) {
    let digit = num % 10;
    reverse = reverse * 10 + digit;
    num = Math.floor(num / 10);
  }
  console.log(originalVal === reverse ? "Palindrome" : "Not Palindrome");
}

isPalindrome(233332);

File: Logical Programs/Palindrome.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Logical_Programs
{
    internal class Palindrome
    {
        public static void IsPalindrome(int num)
        {
            int original = num;
            int reverse = 0;
            while (num > 0)
            {
                int digit = num % 10;
                reverse = reverse * 10 + digit;
                num = num / 10;
            }
            if(original == reverse)
                Console.WriteLine("Palindrome");

        }

        public static void Main()
        {
            IsPalindrome(22312);
        }
    }
}
